### ISSUE TITLE
Cannot download auto-sub for some videos.

### ISSUE THREAD

**Issue by tuanbs** [ytdl-org/youtube-dl#29623]
I cannot download the auto-sub with the following video:
```bash
youtube-dl --write-auto-sub --sub-lang=fr --skip-download "https://www.youtube.com/watch?v=uOuwOc5l1AA"
```

But it works fine with another following video:
```bash
youtube-dl --write-auto-sub --sub-lang=fr --skip-download "https://www.youtube.com/watch?v=R9OCA6UFE-0"
```

Thanks help. Thanks in advance.

**Comment by october262** [comment]
for this - youtube-dl --write-auto-sub --sub-lang=fr --skip-download "https://www.youtube.com/watch?v=uOuwOc5l1AA"  

just change --write-auto-sub to --write-sub

**Comment by tuanbs** [comment]
Hi @october262 , I think you misunderstood this issue. The `--write-sub` is to generate human-generated sub. And the `--write-auto-sub` generates the auto-generated sub. They're both different.

**Comment by kikuyan** [comment]
YouTube doesn't seem to return the auto-sub info for the first video. `--list-subs` shows only `Available subtitles`.
For the second video, `--list-subs` shows `Available automatic captions` and `Available subtitles`.

**Comment by tuanbs** [comment]
@kikuyan Yes that the issue. I've checked the `auto-translate` on YouTube for 1st video and see it has the auto-sub for all languages. Hope this could be fixed in the next version of YouTube-dl.

**Comment by pscj** [comment]
same question. the function of write-auto-sub is not stable in some situation.
